llvm/test/Transforms/Attributor/tag-invariant-loads.ll

```diff
@@ -0,0 +1,324 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -mtriple=amdgcn-amd-amdhsa -passes=attributor %s -S | FileCheck %s --check-prefix=AMDGCN
```

Comment:
Probably needs to be in AMDGPU subdirectory or REQUIRES: amdgpu-registered-target